content/_index.html

```diff
@@ -49,6 +49,6 @@ <h4>Get in touch</h4>
 			<a href="/donate" class="btn btn-success">Donate</a>
 		</div>
 
-		<div>Missing something in the release? Try <a href="/getbuilds">nightly builds</a> instead.</div>
+		<div>To get the latest features and bugfixes, get <a href="/getbuilds">nightly builds</a> instead.</div>
```

Comment:
```suggestion
		<div>For latest features and bugfixes, try <a href="/getbuilds">nightly builds</a> instead.</div>
```
is a bit shorter, and preserves the word "try" which expresses the experimental character of the nightlies better.

Reply:
Just for the information, we have decided on previous wording here: https://chat.reactos.org/reactos/pl/bq36c8yno3ymim8fqoy8rnab6w

Reply:
For latest -> isn't professional English. 

"For the latest" would be better.

Reply:
We do also have to consider, that the text would actually fit there. Space is limited.

Reply:
I think we can omit the "instead" word.
It would look like this:
"For the latest features and bugfixes, try nightly builds."

Reply:
"try our nightly builds?"

well either way im fine with it.

Reply:
"For the latest features and bugfixes, try our nightly builds."

Ok with this?

Reply:
Great to me.